tools/oesign/oesign.c

```diff
@@ -240,6 +269,68 @@ static int _load_config_file(const char* path, config_file_options_t* options)
             options->security_version.value = n;
             options->security_version.has_value = true;
         }
+        else if (strcmp(str_ptr(&lhs), "IsvFamilyID") == 0)
+        {
+            oe_uuid_t id;
+
+            if (options->isv_family_id.has_value)
+            {
+                oe_err(
+                    "%s(%zu): Duplicate 'IsvFamilyID' value provided",
+                    path,
+                    line);
+                goto done;
+            }
+
+            if (str_len(&rhs) > 1)
+            {
+                int rc = uuid_from_string(&rhs, id.b);
+                if (rc != 0)
+                {
+                    oe_err(
+                        "%s(%zu): bad value for 'IsvFamilyID': %s, rc=%d",
+                        path,
+                        line,
+                        str_ptr(&rhs),
+                        rc);
+                    goto done;
+                }
+            }
+
+            memcpy(&options->isv_family_id.value, &id, 16);
```

Comment:
Use sizeof(...) as the 3rd parameter to memcpy(). Same applies to all memcpy()s.

Reply:
Done